### ISSUE TITLE
Call connects but ignores lightning earbuds

### ISSUE THREAD

**Issue by TubaOrNotTuba** [wireapp/wire-ios#1656]
Wire version 3.5 (2945) for iOS fails to recognize when lightning earbuds are plugged in when calls are made and sends sounds through the speakerphone.  One must unplug and replug the lightning earbuds before it switches.  This has happened for several builds of iOS and at least the last couple of builds of Wire for iOS.  It's not my earbuds (OEM Apple) nor my iPhone (7 Plus 256, Verizon, iOS 11.2.5) because neither WhatsApp, Signal, Telegram, Facebook Messenger, Facetime, nor the phone's own dialer exhibit this issue.

This issue, along with Wire's refusal to honour the iPhone's volume settings makes for making calls in quiet places very problematic and unprofessional.

**Comment by mikeger** [comment]
Hey @jspittka ! Is it something handled in 3.6?

@TubaOrNotTuba thanks for the bug report, can you by any chance check on the current AppStore (3.6) ?

**Comment by TubaOrNotTuba** [comment]
Yes, I have 3.6, made no difference. If you connect a pair of OEM Lightning ear buds and initiate a video call, Wire sends audio out the phones speakers not the ear buds.  One must unplug and replug the ear buds to get the phone to silence.

**Comment by malicioustoker** [comment]
Appears to be fixed in TestFlight version 3.7 (2966), but will continue testing

**Comment by mikeger** [comment]
Should be fixed in 3.7, I am going to close the ticket when the version 3.7 is released.
